src/qcommon/files.cpp

```diff
@@ -3281,6 +3327,10 @@ static void FS_Startup( const char *gameName ) {
 	assetsPath = Sys_DefaultAssetsPath();
 	fs_assetspath = Cvar_Get("fs_assetspath", assetsPath ? assetsPath : "", CVAR_INIT | CVAR_VM_NOWRITE);
 
+	assetsPathJKA = Sys_DefaultAssetsPathJKA();
+	fs_assetspathJKA = Cvar_Get("fs_assetspathJKA", assetsPathJKA ? assetsPathJKA : "", CVAR_INIT | CVAR_VM_NOWRITE);
```

Comment:
I think the reason why these cvars (fs_assetspath, fs_gamepath etc.) don't use camelCase is because +set startup arguments are case-sensitive. whatever.

Reply:
As `Com_Init` calls `Com_StartupVariable( NULL );` before initializing the filesystem (https://github.com/Daggolin/jk2mv/blob/0d68927807597aabad81e061d54c36d07a1e8fd9/src/qcommon/common.cpp#L2473) all `+set` arguments are already set as cvars when the filesystem gets initialized. The existing case-sensitive calls to `Com_StatupVariable` in `FS_InitFilesystem` shouldn't even be required. Due to that, it doesn't matter.

I renamed the cvars for consistency anyway.